device_parser.c

```diff
@@ -210,6 +210,8 @@ static void parse_board(struct device_parser *dp)
 			dev->usb_always_on = !strcmp(value, "true");
 		} else if (!strcmp(key, "reset_sequence")) {
 			parse_reset_sequence(dev, value);
+		} else if (!strcmp(key, "release_key_on_fastboot_detect")) {
+			dev->release_key_on_fastboot_detect = true;
```

Comment:
Main question: shouldn't this be a default behaviour? IOW, is there a reason to hold fastboot key longer than necessary?

Reply:
I don't think so no, currently it just holds the fastboot key for 1 second which is enough for dev boards but on some production devices they haven't actually reached the fastboot detection point yet.

I'll make it the default.